0.preprocessing_data/local_preprocessing.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+conda activate gff_preprocessing_env
+
+jupyter nbconvert --to=script --FilesWriter.build_directory=scripts/ notebooks/*.ipynb
+
+cd scripts/ || exit
+
+python 0.update_file_structure.py --HPC False
```

Comment:
```suggestion
python 0.patient_specific_preprocessing.py --HPC False
python 1.update_file_structure.py --HPC False
```
Looks like this would be the correct scripts based on the hpc file.